src/main/java/uk/gov/hmcts/reform/civil/model/defaultjudgment/DisposalHearingBundleDJ.java

```diff
@@ -6,12 +6,14 @@
 import lombok.NoArgsConstructor;
 import uk.gov.hmcts.reform.civil.enums.dj.DisposalHearingBundleType;
 
+import java.util.List;
+
 @Data
 @Builder(toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor
 public class DisposalHearingBundleDJ {
 
     private String input;
-    private DisposalHearingBundleType type;
+    private List<DisposalHearingBundleType> type;
```

Comment:
As we are changing this type to List, how we will manage the existing data in prod which is not of type List? Are we updating/migrating the existing data to List for this field?